$NetBSD: patch-tests_connection__reset__test,v 1.1 2025/05/06 15:49:16 schmonz Exp $

Allocate enough buffer for header sizes seen in common webservers.
https://github.com/renaudallard/sniproxy/commit/1a504817462209ecaa2f62ff925b58852e22f564

--- tests/connection_reset_test.orig	2024-07-02 15:36:27.000000000 +0000
+++ tests/connection_reset_test
@@ -42,8 +42,8 @@ sub simple_server($) {
                       "\r\n");
 
         # Send a bunch of data (more that will be buffered by the kernel
-        for (my $i = 0; $i < 4096; $i++) {
-            $client->send('x' x 4096);
+        for (my $i = 0; $i < 16384; $i++) {
+            $client->send('x' x 16384);
         }
         $client->close();
         exit;
@@ -76,9 +76,9 @@ sub bad_client($) {
                   "\r\n");
 
     my $buffer;
-    $socket->recv($buffer, 4096);
-    $socket->recv($buffer, 4096);
-    $socket->recv($buffer, 4096);
+    $socket->recv($buffer, 16384);
+    $socket->recv($buffer, 16384);
+    $socket->recv($buffer, 16384);
 
     $socket->close();
 
